File: QuoteQuiz.BLL/Models/UserAnswerModel.cs
using System;
using System.Collections.Generic;
using System.Text;

namespace QuoteQuiz.BLL.Models
{
    public class UserAnswerModel
    {
        public Guid Id { get; set; }
        public Guid? QuoteId { get; set; }
        public Guid? AnswerId { get; set; }
        public Guid? UserId { get; set; }
        public bool IsCorrect { get; set; }
        public UserModel User { get; set; }
        public QuoteModel Quote { get; set; }

        public AnswerModel Answer { get; set; }
    }
}

File: QuoteQuiz.Web/ClientApp/src/app/services/user-achievements/models/UserAnswerModel.ts
import { QuoteModel } from '../../quote/models/QuoteModel';
import { UserModel } from './UserModel';
import { AnswerModel } from '../../answer/models/AnswerModel';

export class UserAnswerModel {
    id: string;
    quoteId: string;
    answerId: string;
    userId: string;
    isCorrect: boolean;
    user: UserModel;
    quote: QuoteModel;
    answer: AnswerModel;
}
